### DISCUSSION TITLE
Option "pik_Kp", can you help me

### DISCUSSION THREAD

**Discussion by QuentinOnFire** [dz0ny/klipper-prusa-mk3s#13] [Q&A] [answered]
Hello, i start on klipper and, i have the error Option 'pid_Kp' in section 'extruder' must be specified, can you help me to resolve this ?
Thanx You

**Comment by dz0ny** [comment, marked as answer]
Add
```
[extruder]
pid_Kp: 23.862
pid_Ki: 1.020
pid_Kd: 139.595
```

then run [PID_CALIBRATE HEATER=extruder TARGET=170](https://www.klipper3d.org/Config_checks.html#calibrate-pid-settings) .Klipper recently made this a bit of chicken and the egg problem.

**Comment by scooter12** [comment]
I ran the calibration on the heat_bed as well and stored the results in the main printer.cfg file. I noticed there are no other heat_bed settings anywhere else in the config. Is that expected?

**Reply by dz0ny** [reply]
Yes, as they are pulled as components from https://github.com/dz0ny/klipper-prusa-mk3s/blob/main/mk3s/einsy-rambo.cfg. so basically whatever you put in printer.cfg and is defined elsewhere you override the config.

**Reply by scooter12** [reply]
Thank you for such a quick reply. Happy Holidays!
